assets/queries/openAPI/2.0/unused_parameter_definition/query.rego

```diff
@@ -0,0 +1,21 @@
+package Cx
+
+import data.generic.openapi as openapi_lib
+
+CxPolicy[result] {
+	doc := input.document[i]
+	openapi_lib.check_openapi(doc) == "2.0"
+
+	params := doc.parameters[name]
+	[path, value] := walk(doc)
+	ref := value["$ref"]
+	count({x | ref == sprintf("#/parameters/%s", [name]); x := ref}) == 0
```

Comment:
I think this check could be moved to library.rego for openAPI, since it will be used for other fields